packages/react-components/react-virtualizer/stories/Virtualizer/Dynamic.stories.tsx

```diff
@@ -37,7 +37,7 @@ export const Dynamic = () => {
   const [flag, toggleFlag] = React.useState(false);
   const styles = useStyles();
   const childLength = 1000;
-  const timeoutRef = useRef<NodeJS.Timeout>();
+  const timeoutRef = useRef<ReturnType<typeof setTimeout>>();
```

Comment:
there is no Node global types present in stories within Web Browser. this is a leaking globals from our tsconfig setup. while this doesn't complain atm it will fail once we migrate to https://github.com/microsoft/fluentui/pull/31086